# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sane-frontends/hot-fix.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur sane-frontends-1.0.14-orig/src/gtkglue.c sane-frontends-1.0.14/src/gtkglue.c
--- sane-frontends-1.0.14-orig/src/gtkglue.c	2010-07-23 16:13:33.254914203 +0000
+++ sane-frontends-1.0.14/src/gtkglue.c	2010-07-23 16:14:43.078912232 +0000
@@ -1476,8 +1476,13 @@
 	  || opt->type == SANE_TYPE_GROUP || !dialog->element[i].widget)
 	continue;
 
+#ifdef SANE_CAP_ALWAYS_SETTABLE
       if (!(opt->cap & SANE_CAP_ALWAYS_SETTABLE))
 	gtk_widget_set_sensitive (dialog->element[i].widget, sensitive);
+#else
+      gtk_widget_set_sensitive (dialog->element[i].widget, sensitive);
+#endif
+
     }
 }
 
